libs/watt/package/picker/timepicker/watt-timepicker.component.scss

```diff
@@ -11,7 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-@use "../shared/picker" as *;
+@use "../__shared/picker" as *;
```

Comment:
Question: Have to ask, why the __

Reply:
It is a "limitation" of ng-packagr, where you cannot share code between secondary entrypoints without exposing that shared code as another secondary entrypoint. So the `__` is just a way to tell consumers of the package to not use this entrypoint (in other words, consider it internal).